Sample/RRFPSBar.m

```diff
@@ -30,6 +30,8 @@
 #import "RRFPSBar.h"
 #import <QuartzCore/QuartzCore.h>
 
+// NOTE - Zev Eisenberg - this file has been edited manually since the 0.0.2 release on GitHub.
```

Comment:
💅

Reply:
It's extra confusing because CocoaPods never got 0.0.2; it's stuck on 0.0.1.